optaplanner-examples/src/main/java/org/optaplanner/examples/machinereassignment/swingui/MrMachinePanel.java

```diff
@@ -177,42 +161,4 @@ private void updateTotals() {
         numberOfProcessesLabel.setEnabled(used);
     }
 
-    private class MrProcessAssignmentListDialog extends JDialog {
```

Comment:
Is this necessary? The dialog seems to provide some useful information (if enabled).

Reply:
The dialog was never enabled. Not since this example was created a decade ago was this dialog enabled. This has always been dead code, Geoffrey probably wanted to use it later and did not.